modules/gapi/src/compiler/gcompiler.cpp

```diff
@@ -259,15 +259,17 @@ cv::gimpl::GCompiler::GCompiler(const cv::GComputation &c,
                                                       // (no compound backend present here)
     m_e.addPass("kernels", "check_islands_content", passes::checkIslandsContent);
 
-    m_e.addPassStage("meta");
-    m_e.addPass("meta", "initialize",   std::bind(passes::initMeta, _1, std::ref(m_metas)));
-    m_e.addPass("meta", "propagate",    std::bind(passes::inferMeta, _1, false));
-    m_e.addPass("meta", "finalize",     passes::storeResultingMeta);
-    // moved to another stage, FIXME: two dumps?
-    //    m_e.addPass("meta", "dump_dot",     passes::dumpDotStdout);
-
-    // Special stage for backend-specific transformations
-    // FIXME: document passes hierarchy and order for backend developers
+    if (!m_metas.empty()) 
```

Comment:
Please put a comment here that it is a valid situation when a graph is compiled for streaming

Reply:
Comment is added